reo/validators.py

```diff
@@ -1162,18 +1161,19 @@ def check_special_cases(self, object_name_path, template_values=None, real_value
                     # check if user intended to run CHP and supplied sufficient pararmeters to run CHP
                     if user_supplied_chp_inputs:
                         required_keys = prime_mover_defaults_all['recip_engine'].keys()
-                        filtered_values = {k: real_values.get(k) for k in required_keys}
+                        filtered_values = {k: real_values.get(k) for k in required_keys if k not in ['prime_mover', 'size_class']}
+                        missing_defaults = []
                         for k,v in filtered_values.items():
                             if v is None:
-                                self.input_data_errors.append('No prime_mover was input so all cost and performance parameters must be input. \
-                                    CHP is missing a value for the {} parameter'.format(k))
-
+                                missing_defaults.append(k)
+                        if len(missing_defaults) > 0:                               
```

Comment:
@Bill-Becker I believe this addresses what we talked about, please verify

Reply:
Yep, looks right.